en/docs/try-out/account-and-transaction-flow.md

```diff
@@ -1,7 +1,8 @@
 This document provides step by step instructions to deploy, subscribe, and invoke the Account and Transaction API. 
 
 !!! tip
-    When the TPP provides an Account Information Service as an online service, the TPP is known as an Account Information Services Provider (AISP).
+    When the Accredited Data Recipient provides an Account Information Service as an online service, the Accredited Data Recipient
+    is known as an Account Information Services Provider (AISP).
```

Comment:
we need to remove this note as AISP and this usage is specific only to UK and Berlin spec (not for CDS)